src/main/java/frc/robot/utilities/MotorConfig.java

```diff
@@ -19,9 +19,12 @@
     private NeutralMode neutralMode;
     private double voltageCompSaturation;
     private SupplyCurrentLimitConfiguration currentLimitConfig;
-    private FeedbackDevice feedbackDevice;
-    private RemoteSensorSource remoteSensorSourceType;
-    private int remoteSensorSourceDeviceId;
+    private FeedbackDevice primaryFeedbackDevice;
+    private FeedbackDevice secondaryFeedbackDevice;
+    private RemoteSensorSource remoteSensorSource0Type;
+    private RemoteSensorSource remoteSensorSource1Type;
+    private int remoteSensorSource0DeviceId;
+    private int remoteSensorSource1DeviceId;
```

Comment:
why would we be using 2?

Reply:
if we want the angle motor to have its integrated sensor _and_ the mag encoder, we need to use two feedback devices